### ISSUE TITLE
rebuild it to jar file But the navigation can't show it 

### ISSUE THREAD

**Issue by adasdasdadweq** [cvtienhoven/graylog-plugin-aggregates#30]
Dear
    Happy new Years, thanks for ur plugin, it's very useful. But i got some probleam when  I tried to modify something with this plugin , I test it find when i loaded it from module way with graylog-server. And I packaged it to jar file , I put it in the plugin file , and restart the graylog. But I can't see the naviagtion tab: Aggregates from the top. There is a error info:   Cannot read property 'call' of undefined with js file . I think there is some wrong with the graylog-web-plugin I built with.  So i wondered which version u built with this plugin, is that 2.3.0-rc.1 and did i need to changed something in the package.json in the  manifest directory, I had tried it 2 days, test a lot of version but got failure, hope u can help with this , thx!

**Comment by adasdasdadweq** [comment]
hey
I tried to  use 2.3.0-rc.1 to rebuild it , but still got the same problem .
so is anyone met this problem before , I think is something wrong with the graylog2-web-plugin, or some steps were missing.
I pull a new repos (graylog2-server, and this one ), changed the server repos to 2.3.0.rc-1 tag , then use mvn compile , all the info seems to be fine , after that , i cd to this repos ,and use npm install before mvn package ,jar can be made , but can't work on the docker graylog server , the error info is below ,hope someone can help, thx!!!!
`TypeError: Cannot read property 'call' of undefined
    at t (vendor.552834c48b86209e305c.js:1)
    at Object.<anonymous> (module.js from dll-reference __vendor:1)
    at t (bootstrap e653f92b62d42ceb8ed5:19)
    at Object.<anonymous> (plugin.org.graylog.plugins.aggregates.AggregatesPlugin.e653f92b62d42ceb8ed5.js:55307)
    at t (bootstrap e653f92b62d42ceb8ed5:19)
    at URI.js:18
    at Object.<anonymous> (URI.js:21)
    at t (bootstrap e653f92b62d42ceb8ed5:19)
    at Object.<anonymous> (webpack-entry.js:1)
    at t (bootstrap e653f92b62d42ceb8ed5:19)
    at Object.<anonymous> (index.jsx:2)
    at t (bootstrap e653f92b62d42ceb8ed5:19)
    at bootstrap e653f92b62d42ceb8ed5:62
    at bootstrap e653f92b62d42ceb8ed5:62`

**Comment by cvtienhoven** [comment]
Hey, thanks for you input. Well, setting up the development environment can be a bit of a pain in the ass :) 
I used the graylog-project repo as a starting point in conjunction with the graylog-project-cli to bootstrap the first part of the skeleton. You might want to dive into that to create a proper build environment. Btw, I adapted this to version 2.4.3 and it should be compatible with Graylog 2.4.x now.
